boot-qemu.py

```diff
@@ -216,10 +234,31 @@ def __init__(self):
 
         self._initrd_arch = self._qemu_arch = 'x86_64'
 
+    def supports_efi(self):
+        return True
+
     def run(self):
         if not self.use_kvm:
             self._qemu_args += ['-cpu', 'Nehalem']
 
+        if self.efi:
+            usr_share = Path('/usr/share')
+            ovmf_locations = [
+                Path('edk2/x64/OVMF_CODE.fd'),  # Arch Linux (current), Fedora
+                Path('edk2-ovmf/x64/OVMF_CODE.fd'),  # Arch Linux (old)
+                Path('OVMF/OVMF_CODE.fd'),  # Debian and Ubuntu
+            ]
+            self._efi_img = utils.find_first_file(usr_share, ovmf_locations)
+
+            ovmf_vars_locations = [
+                Path('edk2/x64/OVMF_VARS.fd'),  # Arch Linux and Fedora
+                Path('OVMF/OVMF_VARS.fd'),  # Debian and Ubuntu
+            ]
+            ovmf_vars = utils.find_first_file(usr_share, ovmf_vars_locations)
+            self._efi_vars = Path(BOOT_UTILS, 'images', self.initrd_arch,
+                                  ovmf_vars.name)
+            shutil.copyfile(ovmf_vars, self._efi_vars)
```

Comment:
why does boot-qemu.py copy a file at runtime?

Reply:
It is a `/usr/share` file so it would require root to modify plus it is a blank EFI variables file so it should not be updated at the main location every time that the VM runs. Want a comment?

Reply:
Done: 2ca85ba5d2bfd0fd4226ca7b06993df20694b7b5